app/views/shelters/show.html.erb

```diff
@@ -19,6 +24,6 @@
   <% if review.image != nil %>
     <%= image_tag review.image %>
   <% end %>
```

Comment:
Will need to add image size max

Reply:
Yes. The seed file has a review image which some would say is too big